### DISCUSSION TITLE
Welcome to Seg3D Discussions!

### DISCUSSION THREAD

**Discussion by dcwhite** [SCIInstitute/Seg3D#390] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by massisenergy** [comment]
Hello,

It's a nice software. I am trying out different Open-source 3D image processing softwares in my new M1Pro. Surprisingly, only seg3D is performing well without issues ATM. So I thank the developers 👍🏽 

I don't have a specific problem with the image processing but I might soon, as I delve deeper in my image processing pipeline with morphological and greyscale operations. 

My only question is – is there a dedicated forum to discuss seg3D issues or ask for help, like the [3DSlicer discourse community](https://discourse.slicer.org) or the [Image.sc forum](https://forum.image.sc)?

**Reply by massisenergy** [reply]
If we have such dedicated forums, then it will surely increase the outreach even more, I believe. I tried sending a mail to `sympa@lists.sci.utah.edu`, [as mentioned here](https://www.sci.utah.edu/cibc-software/seg3d.html) but it's not working.

**Reply by jessdtate** [reply]
Thanks for posting on the forum! 

you can ask questions about using Seg3D either on this forum or on the seg3d mailing list.  Since you are having problems  subscribing to the list, perhaps you can tell me if you receive a response when you send email to sympa@lists.sci.utah.edu with `subscribe seg3d`  in the body of message.
